### ISSUE TITLE
Integration tests for an authenticated user

### ISSUE THREAD

**Issue by RafterGit** [raft-tech/TANF-app#310]
**Description:**
Expanding integration and accessibility checks to mock the access of admin users to detect any issues with expected behaviors 

**Acceptance Criteria:** 

- [ ] Functioning integration tests that pass through login
- [ ] Testing Checklist has been run and all tests pass

**Tasks:**

- [ ] Create a stub for every django endpoint we expect the client to be able to hit
- [ ] During the normal authorization process, instead of redirecting to login.gov, the target will be the default landing page after login (currently this is `/edit-profile`)
- [ ] `/v1/auth_check` will hand the authorized flag over to redux state
- [ ] We can view app from with in our tests as if we'd authorized through login.gov

**Notes:**

- Based on discussion w/ Lauren: this has been de-prioritized for OFA MVP
- Research will have to be conducted in order to implement OTP solutions for Cypress and Pa11y #377
- Is not tied to specific control but would be useful to demo to OCIO
- Security Controls: SC-04, AC-02, AC-03

**Supporting Documentation:**
_Please include any relevant log snippets/files/screen shots_

**Open Questions:**
_Please include any questions or decisions that must be made before beginning work or to confidently call this issue complete_ 

-

**Comment by shubhi-raft** [comment]
This issue has been de-prioritized for OFA MVP

**Comment by riatzukiza** [comment]
after 1044, we have a path to running integration tests.

**Comment by amilash** [comment]
Related to #1044. not actually blocked by.

**Comment by abottoms-coder** [comment]
Due to the scope of this ticket, I have removed it from the AMS epic and will be converting to an epic for subtasks. Will create a follow-on ticket for testing specifically AMS.

**Comment by ADPennington** [comment]
@abottoms-coder can we discuss the goal and priority of this? for example, is this something we should prioritize for release 1? topic for dev sync 11/10. cc: @valcollignon

**Comment by valcollignon** [comment]
This issue is to improve dev experience, reduce bugs, and general confidence moving forward, but it is not needed for initial production. Per @abottoms-coder / dev sync 11.10.21. Moving into V3 for now, but could be V4.

**Comment by ADPennington** [comment]
I will create a ticket to start generating a list of items to test per page for the QA team cc: @valcollignon
